pheps/phep-9999.md

```diff
@@ -0,0 +1,118 @@
+Requires: 1
+Created: 28-Nov-2023
+Post-History: 28-Nov-2023
+```
+
+# Abstract
+<a name="abstract"></a>
+This PHEP provides a template to make it easier to write a PHEP-1 compliant PHEP. It does not establish or enforce any standard. Each section contains explanatory text for how to write that section in an actual PHEP.
+
+The majority of the text should be replaced with the text of your proposed PHEP. See [Specification](#specification) for further directions on using this template.
+
+This section of your PHEP should include a short description of the issue being addressed.
+
+# Motivation
+<a name="motivation"></a>
+Discussions in the drafing of PHEP 1 indicated a desire to have a template to make it easier for potential PHEP writers to comply with the format, address the required elements, and avoid the "blank page" problem.
+
+The motivation must clearly explain why the existing standards are inadequate to address the problem that the PHEP solves. This can include collecting documented support for the PHEP from important projects in the PyHC ecosystem.
+
+# Rationale
+<a name="rationale"></a>
+The template model should help make the initial drafting of a PHEP a fairly simple copy-paste-replace operation. Although in theory any PHEP could be taken as a template, each PHEP is unique and it is easier to have one that spans the set of expected sections, while minimizing additional information.
+
+The rationale fleshes out the specification by describing why particular design decisions were made. It should describe alternate designs that were considered and related work.
+
+The rationale should provide evidence of consensus within the community and discuss important objections or concerns raised during discussion.
+
+# Specification
+<a name="specification"></a>
+Use this PHEP by copying it to a new file, usually named `phep-9999.md`, and editing it. (See PHEP 1 for details on the process.) The text is in Markdown (specifically [CommonMark](https://spec.commonmark.org/)) and can be directly edited by any text editor. You can check the rendering of text fragments [here](https://spec.commonmark.org/dingus/), and editors with explicit Markdown support include:
+
+* [ReText](https://github.com/retext-project/retext)
+* [PyCharm](https://www.jetbrains.com/help/pycharm/markdown.html)
+
+Start by updating the preamble as follows:
+
+* **PHEP** number should be 9999 until a new number is assigned by the editors.
+* **Author** should be updated with your name and (ideally) email address and ORCID.
+* **Discussions-To** is the URL of your pull request once it is opened (so this usually is not updated until the PR is open).
+* **Status** is Draft
+* **Type** is "Standards Track" if proposing a requirement for PyHC packages; "Informational" for best practices and similar information; "Process" if proposing a means by which PyHC conducts itself (see PHEP 1).
+* **Content-Type** must remain, verbatim `text/markdown; charset=UTF-8; variant=CommonMark`
```

Comment:
We should probably end this bullet with a period too (hopefully people will know not to include it when they copy this content-type verbatim.

Reply:
The end of the "code" markup should help.